src/test/java/uk/gov/pay/ledger/event/service/EventServiceTest.java

```diff
@@ -55,7 +56,7 @@ public void setUp() {
                 .withResourceExternalId(resourceExternalId)
                 .withEventDate(ZonedDateTime.now().minusHours(2L))
                 .toEntity();
-        when(mockEventDao.getEventsByResourceExternalId(resourceExternalId)).thenReturn(List.of(event1, event2));
+        lenient().when(mockEventDao.getEventsByResourceExternalId(resourceExternalId)).thenReturn(List.of(event1, event2));
```

Comment:
Why is this needed now?

Reply:
Mockito for Junit5 complains about unused stubbing and tests fails, as this stub is only used for few tests. It might be that previous version doesn't require strict stubs or has different rules for it. Going to look into more detail